### ISSUE TITLE
Accessing $wpdb's public vars is is flagged as a database query

### ISSUE THREAD

**Issue by JDGrimes** [WordPress-Coding-Standards/WordPress-Coding-Standards#178]
For example:

```php
$site_id = $wpdb->siteid
```

This was unexpected.

**Comment by JDGrimes** [comment]
The PR was merged and released in 0.3.0. So I guess this is fixed.
